iOS/Flows/Archives/Conversation/ConversationContentView.swift

```diff
@@ -44,11 +44,6 @@ class ConversationContentView: View {
     }
 
     func configure(with item: Conversation) {
-
-        defer {
-            self.currentItem = item
-        }
-
```

Comment:
The Xcode 13 simulators are all kinds of messed up so I'm using the beta for now. Unfortunately the beta doesn't know how to handle this defer statement so I'm just putting it's body at the end of the function.